ofrak_patch_maker/ofrak_patch_maker/patch_maker.py

```diff
@@ -574,6 +575,7 @@ async def inject_patch(
         :param resource: OFRAK firmware resource
         :param verbose: prints patch section sizes and destinations at the effective logging level.
         """
+        self.logger.warn("This function is being deprecated!!")
```

Comment:
I'd be comfortable just deleting this method.

If you want to keep it around during the transition, then I'd suggest moving the ofrak imports into the body of the method (and making `resource: "Resource"` typehint in signature.

Reply:
+1, I don't love keeping this method around but I can see how it helps the transition. I really like the idea of moving the imports into the body of the function.

Reply:
Done.